_pages/en/strategy-build-vs-buy.md

```diff
@@ -87,125 +63,136 @@ All ESDC personnel involved in the exploration of IT Solutions and in IT investm
+The benefits to the adoption of this Strategy are expected to reduce risk [^1], reduce technical debt over time, increase client satisfaction, and increase overall confidence in the department and its staff, for both net new or enhancements to IT Solutions.
 
-This strategy capitalizes on existing IT initiatives (such as the [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx)) to compliment them by adding attention to them, as well as complimenting them with new activities.
+This strategy capitalizes on existing IT initiatives (such as the [IITB Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx)) to complement them, as well as supporting them with new activities (see [Coherent set of actions](#coherent-set-of-actions)).
 
-This strategy's goal is to define a Build Versus Buy Assessment Framework, and provide a roadmap in getting to the adoption of this framework.
+This strategy's goal is to clarify the application of the GoC Mandatory Procedures for [Enterprise Architecture Assessments](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) and [on Application Programming Interfaces](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604) when obtaining IT Solutions within ESDC IITB context, and provide a roadmap in getting to the adoption of this framework.
 
 More details in [Appendix A](#appendix-a---business-case-diagnostic).
 
 ## Guiding Policy
 
-The following policy reflects the decision adopted by the **CIO of ESDC** (approval by CIO not yet obtained) when deciding whether an IT solution should be built or bought.
-Each policy statement is a declaration of that decision and has received the endorsement of its associated area of governance body (endorsements not yet obtained, see section [coherent set of actions](#coherent-set-of-actions)).
+The following policy reflects the decision adopted by the **CIO of ESDC** (approval by CIO not yet obtained) when assessing the most appropriate way of obtaining an IT solution, whether by adopting, buying or building it.
+Each policy statement is a declaration of that decision and has received the endorsement of its associated area of governance body (endorsements not yet obtained, see [Coherent set of actions](#coherent-set-of-actions)).
 
-This policy becomes active when IT Solutions are to be obtained, whether delivered by ESDC or by a third-party provider.
-Once active, all teams involved in the project, and the IT products involved in the IT solution, must comply with this guiding policy.
+This policy becomes active when an ESDC initiative requires for an IT Solutions to be obtained, whether delivered by ESDC or by a third-party provider.
```

Comment:
Proposed change to this line "This policy becomes active when an ESDC initiative requires an IT solution to be obtained.."

Reply:
This policy becomes active when an ESDC initiative requires for an IT Solutions to be obtained, whether delivered by ESDC or by a third-party provider.
==>
This policy becomes active when an ESDC initiative requires **that** an IT Solutions be obtained, whether delivered by ESDC or by a third-party provider.